src/api/cartocss.js

```diff
@@ -140,22 +170,30 @@ function guessMap(sql, tableName, column, stats) {
       css = CSS.choropleth(stats.equalint, tableName, columnName, geometryType, ramps.red);
     } else {
       if (stats.dist_type === 'J') {
-        css = CSS.choropleth(stats.headtails, tableName, columnName, geometryType, ramps.red);
+        css = CSS.choropleth(stats.headtails, tableName, columnName, geometryType, ramps.green);
       } else {
         var inverse_ramp = (_.clone(ramps.red)).reverse();
         css = CSS.choropleth(stats.headtails, tableName, columnName, geometryType, inverse_ramp);
       }
     }
-  
+
   } else if (stats.type == 'string') {
 
-      wizard   = "category";
-      css      = CSS.category(stats.hist.slice(0, ramps.category.length).map(function(r) { return r[0]; }),tableName, columnName, geometryType);
+    wizard   = "category";
+    css      = CSS.category(stats.hist.slice(0, ramps.category.length).map(function(r) { return r[0]; }), tableName, columnName, geometryType);
+    metadata = CSS.categoryMetadata(stats.hist.slice(0, ramps.category.length).map(function(r) { return r[0]; }), tableName, columnName, geometryType);
 
-    }
+  } else if (stats.type === 'date') {
+    wizard = "torque";
+    css = CSS.torque(stats, tableName);
+  }
 
   if (css) {
-    return { sql: sql, css: css, geometryType: geometryType, column: columnName, bbox: bbox, stats: stats, type: type, wizard: wizard  };
+    if (metadata) {
+      return { sql: sql, css: css, metadata: metadata, geometryType: geometryType, column: columnName, bbox: bbox, stats: stats, type: type, wizard: wizard  };
```

Comment:
I'd not call it wizard, in this context is more like "visualizationType" than wizard